core/templates/dev/head/pages/collection_player/collection_player.html

```diff
@@ -58,6 +71,113 @@
         left: 42px;
         position: absolute;
       }
+      @media only screen and (max-width : 600px)
+      {
```

Comment:
Here and elsewhere, follow the existing style of putting the { at the end of the previous line.